contrib/builder/arm/deb/debian-jessie/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM ioft/armhf-debian:jessie
```

Comment:
wondering; I know this is a temporary solution, but can we / should we use a more official image for this? At least something under the `docker/` namespace?

ping @tianon for ideas/suggestions